From 1b241aa8572ee8cd4131fffca838b6bbdf5a7b5e Mon Sep 17 00:00:00 2001
From: Christoph Reiter <reiter.christoph@gmail.com>
Date: Wed, 12 Feb 2025 22:29:34 +0100
Subject: [PATCH 122/N] fixup! add python config sh

OSTYPE has changed in MSYS2, so check for both the old and
the new value:
https://github.com/msys2/MSYS2-packages/commit/076c4a94a618413c8b22bed40ee13649434f5145
---
 Misc/python-config.sh.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Misc/python-config.sh.in b/Misc/python-config.sh.in
index e0e048a..195c317 100644
--- a/Misc/python-config.sh.in
+++ b/Misc/python-config.sh.in
@@ -30,7 +30,7 @@ installed_prefix ()
     # Since we don't know where the output from this script will end up
     # we keep all paths in Windows-land since MSYS2 can handle that
     # while native tools can't handle paths in MSYS2-land.
-    if [ "$OSTYPE" = "msys" ]; then
+    if [ "$OSTYPE" = "cygwin" ] || [ "$OSTYPE" = "msys" ] ; then
         RESULT=$(cd "$RESULT" && pwd -W)
     fi
     echo $RESULT
